### ISSUE TITLE
[Bug]: Add/Remove Programs version does not match actual version of seDirector.

### ISSUE THREAD

**Issue by asheroto** [seDirector/App#75]
### Checklist

- [X] I have verified no other issues exist related to my problem.
- [X] I have verified this issue is not security related. 
- [X] I have verified that the latest version of [.NET
Framework](https://dotnet.microsoft.com/en-us/download/dotnet-framework)
has been installed.

- [X] I have restarted my computer to confirm this is not a transitory
problem.

### What You Are Seeing?

If you perform an update through seDirector, the version shown in Add/Remove programs does not match the version shown in seDirector.

If you update using the EXE installer, it works fine.

### What is Expected?

Versions to match

### How Did You Get This To Happen?

N/A

### System Details

N/A

### Additional Context

This issue is caused by the uninstall key, example

```
HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{E12A9D9F-A050-4B90-86E3-D876A3657D8B}
```

value name `DisplayVersion`

**Comment by asheroto** [comment]
Consider using Advanced Installer for updater features as well.

**Comment by asheroto** [comment]
Fixed in upcoming version 7.4.4.
